defaults/main.yml

```
@@ -16,6 +16,8 @@ smartd_default_non_smart_settings:
   - lookahead,on
 smartd_default_test_schedule: (L/../../6/01|S/../.././02)
 smartd_default_mail_frequency: diminishing
+smartd_default_mail_script:
+smartd_default_mail_recipients:
```

Can you change this to an empty list so the reader knows of which type the variable should be?

```suggestion
smartd_default_mail_recipients: []
```

---

Corrected.

---

Merged. For future reference: Suggested changes can be accepted in the GitHub UI - that way you don't have to commit the same change manually.